site/sfguides/src/tealium-snowflake-integration/tealium-snowflake-integration.md

```diff
@@ -0,0 +1,471 @@
+author: jmusselwhite-tealium
+id: tealium-snowflake-integration
+summary: This tutorial reviews the process for configuring inbound and outbound Snowflake integrations within Tealium Customer Data Hub
+categories: Getting-Started, Partner-Integrations, Marketing, Architecture-Patterns
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Tealium, Snowpipe Streaming, Staging Tables, Real-Time, CDP, Customer Data Platform, Data Activation
+
+# Collect and Activate Customer Data Using Tealium and Snowpipe Streaming
+
+<!-- ------------------------ -->
+## Overview 
+Duration: 2
+
+## Introduction
+This quickstart guide will review the configuration of the Snowflake Data Source and Snowflake Streaming connectors within Tealium. Tealium's Customer Data Hub acts as a high quality data source and a real-time activation engine for Snowflake. Use Tealium to capture and send low latency customer interaction data to Snowflake for further analysis or to inform AI initiatives, and automate real-time data activation using Snowflake data across Tealium’s 1300+ integration marketplace.
+
+#### Send Data to Snowflake
+- Create a staging table in Snowflake to land the data
+- In Tealium, configure the Snowflake Streaming connector
+- Process the data from the staging table to the appropriate tables in Snowflake
+
+#### Activate Data from Snowflake
+- Identify or create a table or view of data in Snowflake that you want to activate in Tealium
+- In Tealium configure the data source by establishing a connection to Snowflake, creating a query to capture only the data needed, map the event and visitor data to appropriate Tealium data attributes
+- Once the connection is established, data is ingested in near real-time (as fast as 2 seconds) and is activated based on business rules 
+
+
+### What You'll Need
+
+- Tealium Customer Data Hub (EventStream and/or AudienceStream) account
+- Snowflake account
```

Comment:
@jonathan-musselwhite @sfc-gh-lambrosetti Add link to Snowflake signup/signin page.

Reply:
Create Snowflake Account as a link